### PR TITLE
Update instructions for `shiny create`

### LINKED ISSUE #59: Installation page running command outdated
In the page [Installing Shiny for Python](https://shiny.posit.co/py/docs/install.html) page there is a reference to running the code:

`shiny create .`

This command is not correct. The output error message says: "_Error: Got unexpected extra argument (.)_".

### PR DESCRIPTION
**Pull request by wch**
This fixes #59.

### CODE REVIEW

File: docs/comp-r-shiny.qmd [hunk 1]
```diff
@@ -44,7 +44,7 @@ conda install -c conda-forge shiny
 ```
 :::
 
-3.  Call `shiny create .` to create an example shiny app in your directory
+3.  Call `shiny create` to create an example shiny app in your directory
```
Comment (gshotwell) [inline docs/comp-r-shiny.qmd:47]:
```suggestion
3.  Call `shiny create -t basic-app -d .` to create an example shiny app in your directory. You can also call `shiny create` without arguments to select a different template.
```

File: docs/install.qmd [hunk 2]
```diff
@@ -74,7 +74,7 @@ conda update -c conda-forge shiny
 In the same directory you created above, run:
 
 ```bash
-shiny create .
+shiny create
```
Comment (gshotwell) [inline docs/install.qmd:77]:
```suggestion
shiny create -t basic-app -m express -d .
```

**Review by gshotwell — COMMENTED** [review]
I added some suggestions which will replicate the old behaviour. I also think we want to add `--launch-browser` to the run command or change the text (it says that it'll launch a browser but it doesn't currently).

File: docs/install.qmd (hunk 2 above)
Comment (gshotwell) [reply docs/install.qmd:77]: Oh that's a good note, we have a huge amount of documentation about to ship on Express, but for this PR we should probably take it out. Thanks! 

For reference here's the Express documentation PR https://github.com/posit-dev/py-shiny-site

**Comment by wch** [comment]
Thanks for the feedback. I also added the `--launch-browser` option to the instructions.

**Review by robertgv — APPROVED** [review]

**Comment by schloerke** [comment]
Closing as the instructions have been updated since this PR and cover the PR's situations.

https://github.com/posit-dev/py-shiny-site/blob/3e65247453c0a0a13a0d4b59fad6cdb981141f31/docs/install-create-run.qmd

**Pull request closed without merging**